src/calculateRentalCost.js

```diff
@@ -5,6 +5,18 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const cost = 40;
+  let totalCost = cost * days;
+
+  if (days >= 7) {
+    return (totalCost -= 50);
+  }
```

Comment:
Prefer using 'if' with 'return' over 'if else' to simplify the code. Since there's a return inside the 'if', the 'else' is not necessary.